### ISSUE TITLE
[BUG] - Lower Right corner of "Forest"

### ISSUE THREAD

**Issue by sumerik** [damian-pastorini/reldens#63]
Player runs in place, server crashes, and logs show:

TypeError: Cannot set property 'g' of undefined
    at AStarFinder.findPath (/home/ubuntu/reldens-skeleton/node_modules/pathfinding/src/finders/AStarFinder.js:67:17)
    at PathFinder.findPath (/home/ubuntu/reldens-skeleton/node_modules/reldens/packages/world/server/path-finder.js:31:32)
    at PhysicalBody.moveToPoint (/home/ubuntu/reldens-skeleton/node_modules/reldens/packages/world/server/physical-body.js:194:48)
    at RoomScene.onMessage (/home/ubuntu/reldens-skeleton/node_modules/reldens/packages/rooms/server/scene.js:155:28)
    at RoomScene._onMessage (/home/ubuntu/reldens-skeleton/node_modules/colyseus/lib/Room.js:420:18)
    at WebSocket.emit (events.js:198:13)
    at Receiver.receiverOnMessage (/home/ubuntu/reldens-skeleton/node_modules/ws/lib/websocket.js:801:20)
    at Receiver.emit (events.js:198:13)
    at Receiver.dataMessage (/home/ubuntu/reldens-skeleton/node_modules/ws/lib/receiver.js:427:14)
    at Receiver.getData (/home/ubuntu/reldens-skeleton/node_modules/ws/lib/receiver.js:366:17)
INFO - ON-DISPOSE Room: chat
INFO - ON-DISPOSE Room: ReldensForest
INFO - Game Server is going down.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reldens-skeleton@0.1.0 start: node .
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the reldens-skeleton@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-03-22T13_24_35_990Z-debug.log

**Comment by damian-pastorini** [comment]
Hi @sumerik , I've identified the issue, the problem seems to be on the grid creation, which right now is taking the world width and height, so when I try to get the node that's at the world limit it returns undefined instead the last world node. This makes me thing that the getNodeAt index starts at 0 been actually the world block 1, to confirm that I've tried by adding a +1 to the grid limit and after the last node works normally. I'm not getting the issue anymore but I will dig a bit more to see if there's any better solution.
Thanks for reporting the issue! :)
